exercises/ex_atm.py

```diff
@@ -0,0 +1,162 @@
+"""ATM Exercise"""
+
+import sys
+
+FILE_PATH_INDEX = 1  # describe placement of the argument containing file path.
+PASSWORD_INDEX = 0
+BALANCE_INDEX = 1
+END_ATM_SERVICE = "-1"
+
+
+def store_file_content(file_path):
+    """Read file and move atm information to dictionary.
+    The lines of the file will contain user's id, balance and password,
+    with comma's separating them.
+
+    Args:
+        file_path (str): path to atm information file.
+
+    Return:
+        dict. with user's id as keys,
+        and user's password and balance in a list as values.
+    """
+    try:
+        atm_options = {}
+        with open(file_path, 'r') as file_pointer:
+            for line in file_pointer:
+                line_list = line.rstrip().split(',')
+                atm_options.update({line_list[0]: [line_list[1],
+                                                   float(line_list[2])]})
+
+        return atm_options
+
+    except FileNotFoundError:
+        print(f"Can't access the path to the file you have entered:"
+              f"{file_path}")
+
+
+def get_user_id(atm_info):
+    while True:
+        user_id_input = input("Enter costumer ID, to turn off the ATM enter "
+                              "-1 -> ")
+        if user_id_input == END_ATM_SERVICE:
+            atm_info = None
```

Comment:
what?

Reply:
in one of the reviews igal wrote that when i use -1 i need to write it as a constant, in case it will change to stop or something like it. if the user just want to get out he enter -1 the atm dict will be none and the code will stop.

Reply:
its ok to stop the program, but no need to change the dict to none